NOTICE

```diff
@@ -1,3 +1,6 @@
+[firebolt-native-client]
+A release package with sources for building native (c++) client is prepared from the src/cpp directory, which needs to include the associated license file (Apache-2.0).
```

Comment:
Hi @tomasz-blasz : Same comments as PR#470 please (remove line 1, move 2-3 to end and change line 4 to the name of this repo)